packages/flutter_tools/lib/src/build_system/targets/macos.dart

```diff
@@ -45,11 +47,45 @@ Future<void> copyFramework(Map<String, ChangeType> updates,
 const Target unpackMacos = Target(
   name: 'unpack_macos',
   inputs: <Source>[
-    Source.pattern('{CACHE_DIR}/darwin-x64/FlutterMacOS.framework/'),
+    Source.pattern('{CACHE_DIR}/artifacts/engine/darwin-x64/FlutterMacOS.framework/FlutterMacOS'),
```

Comment:
nit: "{CACHE_DIR}/artifacts/engine/darwin-x64/FlutterMacOS.framework/" gets repeated a lot.

Reply:
I added a prefix String to shrink this down